src/Speckle.Objects/GIS/GisLayer.cs

```diff
@@ -0,0 +1,12 @@
+using Speckle.Sdk.Models;
```

Comment:
not sure if we need this extra class - you could use the `layer:collection` class and just add the missing props from crs dynamically

Reply:
Is CRS required for data extraction? it seems like it's only used on send, not receive

Reply:
yes, for GIS data, the data must have the georeference